### ISSUE TITLE
IntelliSense for schedule query rule condition operator recommends "Equals" instead of "Equal"

### ISSUE THREAD

**Issue by StephenWeatherford** [Azure/azure-resource-manager-schemas#1548]
From https://github.com/microsoft/vscode-azurearmtools/issues/1257

```json
{
  "type": "microsoft.insights/scheduledQueryRules",
  "apiVersion": "2020-05-01-preview",
  "name": "<some-name>",
  "location": "<some-location>",
  "properties": {
    // other properties
    "criteria": {
      "allOf": [
        {
          "query": "<kql-query>",
          "timeAggregation": "Count",
          "operator": "Equal",  // <<<<< error here
          "threshold": 0,
          "failingPeriods": {
            "numberOfEvaluationPeriods": 1,
            "minFailingPeriodsToAlert": 1
          }
        }
      ]
    }
  }
}
```

EXPECTED: no errors
ACTUAL: 2 problems:
1) properties.criteria.allOf[0].operator property gives error:
```
Value must be one of the following values: "Equals", "GreaterThan", "GreaterThanOrEqual", "LessThan", "LessThanOrEqual"
```
2) Unhelpful error message when deploying with recommended "Equals":
```
Alert rule payload cannot be empty
```

**Comment by oliverroer** [comment]
Slightly related: I have also experienced issues with `microsoft.insights/scheduledqueryrules` in ARM.
Deploying with apiVersion `2020-05-01-preview` works just fine, but when using apiVersion `2021-02-01-preview`, the deployment will fail with the error `Alert rule payload cannot be empty`.

It seems that this is some general error message that has nothing to do with the underlying problem, which is very frustrating to work with, since it gives us no clue how to resolve it. In our case, I just happened to try downgrading to apiVersion `2020-05-01-preview` and it just magically started working. I have no clue why though... 🤔
